test/contacts.js

```diff
@@ -78,172 +126,232 @@ describe('contacts', function() {
   })
 
   describe('getByEmail', function() {
-    it('should return a contact record based on the email', function() {
-      return hubspot.contacts
-        .getByEmail('testingapis@hubspot.com')
-        .then(data => {
-          expect(data).to.be.a('object')
-          expect(data.properties).to.be.a('object')
+    let email = 'testingapis@hubspot.com'
+    const getContactByEmailEndpoint = {
+      path: `/contacts/v1/contact/email/${email}/profile`,
+      response: { properties: {} },
+    }
+    fakeHubspotApi.setupServer({ getEndpoints: [getContactByEmailEndpoint] })
+
+    before(function() {
+      if (process.env.NOCK_OFF) {
+        return hubspot.contacts.get({ count: 3 }).then(data => {
```

Comment:
Is there a benefit to setting the count to 3 instead of requesting one? It seems we just need the API to return one contact so the contact's email can be used in the test (when nock is off)

Reply:
I guess to make sure we get one with an email? This is defensive so I can change this to 1 👍

Reply:
Ah, is email not a required field on Contact?